### PR TITLE
Post about the 1st public workshop

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I started a post about the last workshop.

Any feedback is welcome 😄 

Bérénice

### CODE REVIEW

File: _events/2019-03-10-first_public_workshop.md [hunk 1]
```diff
@@ -12,8 +13,103 @@ location:
...
 
-Our first public workshop for the [BeerDEcoded project]({% link _projects/beerdecoded.md %}) will be held on the 31st of March in Freiburg.
+On the 31st of March, we hold our first public workshop for the [BeerDEcoded project]({% link _projects/beerdecoded.md %}) in Freiburg. Some students and external people participated to this event: 16 peoples in total, including us.
 
-You want to particpate? Please send us an email at [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}) or join the [mailing list]({{ site.google_group }}).
+{% include event-images.html images="
+  /images/events/2019-03-10-first_public_workshop/DSC_9661.jpg"
+%}
+
+We started the day with short introduction to beer, yeast, DNA, but also lab safety. We organized after some games around pipetting and centrifuge to make the participants more comfortable with both.
```
Comment (teresa-m) [inline _events/2019-03-10-first_public_workshop.md]: We started the day with a short introduction to beer, yeast, DNA and of course lab safety. Then all participants took part in some fun games on pipetting and centrifuging to make them more comfortable with both.

File: _events/2019-03-10-first_public_workshop.md [hunk 2]
```diff
@@ -12,8 +13,103 @@ location:
...
+
+{% include event-images.html images="
+  /images/events/2019-03-10-first_public_workshop/DSC_9563.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9575.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9583.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9592.jpg"
+%}
+
+We extracted DNA of the yeast of a local Black Forest beer, the not filtered Waldhaus.
```
Comment (teresa-m) [inline _events/2019-03-10-first_public_workshop.md]: We extracted yeast DNA of the not filtered Waldhaus, a local Black Forest beer.

File: _events/2019-03-10-first_public_workshop.md [hunk 3]
```diff
@@ -12,8 +13,103 @@ location:
...
+
+We then analyzed the data using Galaxy.
+
+{% include event-images.html images="
+  /images/events/2019-03-10-first_public_workshop/DSC_9613.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9656.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9538.jpg"
+%}
+
+The participants seem to have enjoyed the day!
```
Comment (teresa-m) [inline _events/2019-03-10-first_public_workshop.md]: The participants and we enjoyed the day a lot!

File: _events/2019-03-10-first_public_workshop.md [hunk 4]
```diff
@@ -12,8 +13,103 @@ location:
...
+{% include event-images.html images="
+  /images/events/2019-03-10-first_public_workshop/DSC_9644.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9596.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9598.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9607.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9647.jpg
+  /images/events/2019-03-10-first_public_workshop/DSC_9609.jpg"
+%}
+
+The extraction went well. In parallel, we tried to sequence DNA extracted previously (from another local beer, Rothaus). But we encountered several problems during the library preparation and the sequencing: mistakes in the protocols, reuse of old flowcell, etc. We will improve the protocol to fix the issues.
```
Comment (teresa-m) [inline _events/2019-03-10-first_public_workshop.md]: DNA extraction went well. In parallel, we started our DNA sequencing protocol using a  previously extracted DNA sequence from Rothaus, another local beer. Here we encountered several problems during the library preparation and the sequencing like mistakes in the protocols and reuse of old flowcell. We will improve the protocol to fix the issues.

**Comment by bebatut** [comment]
Nice catch. Done

**Comment by bgruening** [comment]
danke

**Pull request merged**